src/helpers/buildTransactionsSectionsSelector.tsx

```diff
@@ -65,8 +69,18 @@ const buildTransactionsSections = (
       groupTransactionByDate
     );
     sectionedTransactions = Object.keys(transactionsByDate).map(section => ({
-      data: transactionsByDate[section],
-      renderItem: renderItemElement(TransactionCoinRow),
+      data: transactionsByDate[section].map(txn => ({
+        ...txn,
+        accountAddress,
+        mainnetAddress: mainnetAddresses[`${txn.address}_${txn.network}`],
+      })),
+      renderItem: ({ item }: any) => (
```

Comment:
Can you fix any or use unknown instead?

Reply:
Discussed in DMs. We will leave it as is for now.